src/app/core/components/header/header.component.ts

```diff
@@ -1,10 +1,51 @@
 import { Component } from '@angular/core';
 
+export interface IHeaderMenu {
+  name: string;
+  router: string[];
+  srcImg: string;
+}
+
 @Component({
-    selector: 'app-header',
-    templateUrl: './header.component.html',
-    styleUrls: ['./header.component.scss'],
+  selector: 'app-header',
+  templateUrl: './header.component.html',
+  styleUrls: ['./header.component.scss'],
 })
 export class HeaderComponent {
-    constructor() {}
+  iconImagePath = '../../../../assets/images/icons/header/';
+  isAuthorization = true;
+  userPhoto = 'https://ofigenno.com/wp-content/uploads/posts/z/zoi-saldana/1.jpg';
```

Comment:
Don't use links to external media, it's not robust. If you need it, save to assets and take it from there

Reply:
Done